neutrino-custom/base.js

```diff
@@ -169,6 +170,7 @@ module.exports = neutrino => {
             // Required by Bootstrap 4: https://getbootstrap.com/docs/4.0/getting-started/webpack/
             Popper: ['popper.js', 'default'],
             React: require.resolve('react'),
+            PropTypes: require.resolve('prop-types'),
```

Comment:
Perhaps we should replace these globals with `require()` in the files where they are used instead, whilst we're here? :-)